README.md

```diff
@@ -258,16 +264,12 @@ Fill in the details required for this post build action. You need the following
 
 You need to now enter the credentials that lets Jenkins trigger a release with the latest completed build, on your behalf. If you are using VS Team Services, you just need to enter **PAT** with atleast "Release (read, write and execute)" scope. (Refer to this [link](https://www.visualstudio.com/en-us/get-started/setup/use-personal-access-tokens-to-authenticate) to understand how to create PAT). In case you are using TFS, you need to enter the **username** and **password**.
 
-![Add post build action](screenshots/fillFieldsForPostBuildAction.png)
+![Add post build action](https://cloud.githubusercontent.com/assets/15227510/18209903/8e94864a-7153-11e6-852d-c8ff28e38667.png)
```

Comment:
Isnt it better to stick to relative links.. then it works even in a forked repo and not just the main one. And it is also easier to understand which image to update..